packages/core/src/codewhispererChat/tools/listDirectory.ts

```diff
@@ -61,6 +62,18 @@ export class ListDirectory {
         updates.end()
     }
 
+    public requiresAcceptance(): CommandValidation {
```

Comment:
this looks very similar to [this](https://github.com/aws/aws-toolkit-vscode/blob/a32d7ea6bef8acaabcc6d7737a9b7c1668cecd92/packages/core/src/codewhispererChat/tools/executeBash.ts#L199-L208), can we dedupe?